### PR TITLE
Pooler performance optimisations

### PR DESCRIPTION
**Pull request by fenos**
## What kind of change does this PR introduce?

Performance optimisation to pooler

### CODE REVIEW

**Review by pcnc — APPROVED** [review]
LGTM (except for the test pass 👀)

File: src/http/routes/bucket/createBucket.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ const createBucketBodySchema = {
     },
     allowed_mime_types: {
       type: 'array',
+      nullable: true,
       items: { type: 'string' },
-      examples: [['image/png', 'image/jpg']],
```
Comment (inian) [inline src/http/routes/bucket/createBucket.ts:21]: why is this removed @fenos ?

**Review by inian — APPROVED** [review]

File: src/http/routes/bucket/createBucket.ts (hunk 1 above)
Comment (fenos) [reply src/http/routes/bucket/createBucket.ts:21]: added it back
